src/views/Admin/components/CliftonStrengthsUpload/index.js

```diff
@@ -0,0 +1,141 @@
+import {
+  Button,
+  Card,
+  CardHeader,
+  Grid,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import { DateTime } from 'luxon';
+import { useState } from 'react';
+import { gordonColors } from 'theme';
+import GordonLoader from '../../../../components/Loader';
+import SpreadsheetUploader from '../../../../components/SpreadsheetUploader';
+import CliftonStrengthsService from '../../../../services/cliftonStrengths';
+import CliftonStrengthsUploadTemplate from './cliftonStrengthsUploadTemplate.csv';
+
+const headerStyle = {
+  backgroundColor: gordonColors.primary.blue,
+  color: gordonColors.neutral.grayShades[100],
+};
+
+const successResults = ['Success', 'Added', 'Modified'];
+
+const CliftonStrengthsUpload = () => {
+  const [isSpreadsheetUploaderOpen, setIsSpreadsheetUploaderOpen] = useState(false);
+  const [uploadResults, setUploadResults] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmitStrengths = (uploadedData) => {
+    setLoading(true);
+
+    let formattedData = uploadedData.map((cs) => {
+      // The date completed comes out of the
+      // spreadsheet uploader as a number of days since 1/1/1900.
+      let days = cs.DateCompleted;
+      cs.DateCompleted = DateTime.fromFormat('01-01-1900', 'MM-dd-yyyy')
+        .plus({ days: days })
+        .toISODate();
+
+      return {
+        AccessCode: cs.AccessCode,
+        Email: cs.Email,
+        DateCompleted: cs.DateCompleted,
+        Themes: [cs.Theme1, cs.Theme2, cs.Theme3, cs.Theme4, cs.Theme5],
+        Private: false,
+      };
+    });
+
+    CliftonStrengthsService.postCliftonStrengths(formattedData)
+      .then((data) => {
+        setUploadResults(
+          data.map((r, index) => (
+            <TableRow key={index} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
+              <TableCell style={{ fontSize: '1rem' }}>{r.Email}</TableCell>
+              <TableCell style={{ fontSize: '1rem' }}>{r.AccessCode}</TableCell>
+              <TableCell
+                style={{
+                  backgroundColor: successResults.some((s) => s === r.UploadResult)
+                    ? gordonColors.secondary.greenShades.secondary
+                    : gordonColors.secondary.red,
```

Comment:
I think a colored icon would be a more aesthetically pleasing way to distinguish the result than coloring the entire table cell. This is also more consistent with the rest of the site, and more accessible to people with colorblindness.

Reply:
I think this looks good:
![image](https://user-images.githubusercontent.com/43764277/189925383-d8fe6dab-cc25-4c71-8029-64064f94f680.png)